File-Formats/PDF/Working-with-Document-Conversions.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: Working with Document Conversion | Syncfusion
-description: This section explains converting other document Types (Word, Excel, RTF, TIFF, XPS and HTML) into PDF
+title: Working with Document Conversion in Fileformat PDF | Syncfusion
```

Comment:
Add `control` word in meta title - check all places.

Reply:
Resolved